portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -10,9 +27,40 @@
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
+  private final Date startTime = new Date();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    // Calculate server stats
+    Date currentTime = new Date();
+
+    // Convert data to JSON
+    MarcoStats marcoStats = new MarcoStats(20, "Marco");
+    String json = convertToJsonUsingGson(marcoStats);
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+
+    //response.setContentType("text/html;");
```

Comment:
Please get rid of unused code, or if you're planning on using this in the future - consider adding a TODO!